training_record/serializers.py

```diff
@@ -0,0 +1,44 @@
+'''Define how to serialize our models.'''
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+import training_record.models
+
+
+User = get_user_model()
+
+
+class UserSerializer(serializers.ModelSerializer):
```

Comment:
Delete this.
As comment at https://github.com/DLUT-SIE/TMSFTT-BE/pull/10/files#r248509481